src/files.js

```diff
@@ -47,10 +47,20 @@ function readFile(path, fileName) {
  * @param {string} templateFileName
  */
 function generateFileName(newFileName, templateFileName) {
+  let generatedFileName = templateFileName
   if (templateFileName.includes('COMPONENT_NAME')) {
-    return templateFileName.replace(/COMPONENT_NAME/g, newFileName)
+    generatedFileName = templateFileName.replace(/COMPONENT_NAME/g, newFileName)
```

Comment:
@damiansilbergleithcunniff 

What do you think to use an object instead if statement ?
Something like: 
```javascript
{
  COMPONENT_NAME: newFileName,
  component_name: newFileName.toLowerCase(),
...
}
```

And use a reduce function to replace content?
Understand what I mean?